redis_setup.sh

```diff
@@ -15,10 +18,19 @@ sudo add-apt-repository    "deb [arch=amd64] https://download.docker.com/linux/u
 sudo apt-get update
 sudo apt-get install docker-ce
 
+else
+    echo "yesss exist"
```

Comment:
Fix these messages. Please use something professional.

Reply:
ohh sorry! I was using it for checking.

Reply:
```
./redis_setup.sh 
Docker version 18.06.0-ce, build 0ffa825
./redis_setup.sh: line 4: [: too many arguments
Docker is already installed
``` 
Fix this error message